beacon/endpoints/rest/response/info_response_schema.py

```diff
@@ -38,72 +37,21 @@ def build_received_request(qparams):
 
     request = {
         'meta': {
-            'requestedSchemas' : build_requested_schemas(qparams),
+            'requestedSchemas' : [qparams.requestedSchema[0]],
             'apiVersion' : qparams.apiVersion,
         },
     }
 
     return request
 
 
-def build_requested_schemas(qparams):
-    """"
-    Fills the `requestedSchemas` part with the request data
-    It includes valid and invalid schemas requested by the user.
-    """
-
-    requested_schemas = {}
-
-    if qparams.requestedSchemasServiceInfo[0] or qparams.requestedSchemasServiceInfo[1]:
-        requested_schemas['ServiceInfo'] = [s for s, f in qparams.requestedSchemasServiceInfo[0]] + list(
-            qparams.requestedSchemasServiceInfo[1])
-
-    if qparams.requestedSchemasDataset[0] or qparams.requestedSchemasDataset[1]:
-        requested_schemas['Dataset'] = [s for s, f in qparams.requestedSchemasDataset[0]] + list(
-            qparams.requestedSchemasDataset[1])
-
-    return requested_schemas
-
-
-def build_returned_schemas(qparams, func_response_type):
-    """"
-    Fills the `returnedSchema` part with the actual schemas returned in the response.
-    This is the default schema for each type and any valid schema requested by the user.
-    """
-
-    # LOG.debug('func_response_type= %s', func_response_type.__name__)
-
-    returned_schemas_by_response_type = {
-        'build_service_info_response': {
-            'ServiceInfo': [DEFAULT_SCHEMAS['ServiceInfo']] if not qparams.requestedSchemasServiceInfo[0] else []
-                           + [s for s, f in qparams.requestedSchemasServiceInfo[0]],
-        },
-        'build_dataset_info_response': {
-            'Dataset': [DEFAULT_SCHEMAS['Dataset']] if not qparams.requestedSchemasDataset[0] else []
-                       + [s for s, f in qparams.requestedSchemasDataset[0]],
-        },
-    }
-
-    return returned_schemas_by_response_type[func_response_type.__name__] # We let it throw a KeyError
-
-
 def build_error(qparams):
```

Comment:
Que?

Reply:
Currently, I don't know of any (partial) error we may have when calling `/info` or `/datasets` but I left the method just in case...

Reply:
I've removed it.